diff --git a/node_modules/@evershop/evershop/dist/components/frontStore/customer/address/addressForm/AddressForm.js b/node_modules/@evershop/evershop/dist/components/frontStore/customer/address/addressForm/AddressForm.js
index 0da00e1..d30e2b8 100644
--- a/node_modules/@evershop/evershop/dist/components/frontStore/customer/address/addressForm/AddressForm.js
+++ b/node_modules/@evershop/evershop/dist/components/frontStore/customer/address/addressForm/AddressForm.js
@@ -56,7 +56,12 @@ export function CustomerAddressForm({ allowCountries, address = {}, formId = 'cu
                     value: address === null || address === void 0 ? void 0 : address.city,
                     label: _('City'),
                     placeholder: _('City'),
-                    validationRules: []
+                    validationRules: [
+                        {
+                            rule: 'notEmpty',
+                            message: _('City is required')
+                        }
+                    ]
                 },
                 sortOrder: 40
             },
diff --git a/node_modules/@evershop/evershop/dist/lib/webpack/createBaseConfig.js b/node_modules/@evershop/evershop/dist/lib/webpack/createBaseConfig.js
index 0cbe3c3..4c1255a 100644
--- a/node_modules/@evershop/evershop/dist/lib/webpack/createBaseConfig.js
+++ b/node_modules/@evershop/evershop/dist/lib/webpack/createBaseConfig.js
@@ -22,7 +22,7 @@ export function createBaseConfig(isServer) {
             }
         },
         {
-            test: /^[A-Z].*\.js$/,
+            test: /[\\/][A-Z][^\\/]*\.(js|jsx)$/,
             exclude: {
                 and: [/node_modules/],
                 not: [
@@ -101,19 +101,24 @@ export function createBaseConfig(isServer) {
     }
     // Resolve aliases
     const alias = {
-        '@evershop/evershop/components': path.resolve(__dirname, '../../components')
+        '@evershop/evershop/components': path.resolve(__dirname, '../../components'),
+        '@evershop/evershop/lib': path.resolve(__dirname, '..')
     };
     if (theme) {
         alias['@components'] = [path.resolve(theme.path, 'dist/components')];
+        alias['@modules'] = [path.resolve(theme.path, 'dist/modules')];
     }
     else {
         alias['@components'] = [];
+        alias['@modules'] = [];
     }
     // Resolve alias for extensions
     extenions.forEach((ext) => {
         alias['@components'].push(path.resolve(ext.resolve, 'dist/components'));
+        alias['@modules'].push(path.resolve(ext.resolve, 'dist/modules'));
     });
     alias['@components'].push(path.resolve(__dirname, '../../components'));
+    alias['@modules'].push(path.resolve(__dirname, '../../modules'));
     // Resolve alias for core components
     alias['@components-origin'] = path.resolve(__dirname, '../../components');
     // Avoid multiple react instances
diff --git a/node_modules/@evershop/evershop/dist/modules/paypal/api/paypalCreateOrder/[bodyParser]createOrder.js b/node_modules/@evershop/evershop/dist/modules/paypal/api/paypalCreateOrder/[bodyParser]createOrder.js
index 1769f90..0761134 100644
--- a/node_modules/@evershop/evershop/dist/modules/paypal/api/paypalCreateOrder/[bodyParser]createOrder.js
+++ b/node_modules/@evershop/evershop/dist/modules/paypal/api/paypalCreateOrder/[bodyParser]createOrder.js
@@ -139,6 +139,7 @@ export default async (request, response, next) => {
                 };
                 if (billingAddress.province) {
                     address.admin_area_1 = billingAddress.province.split('-').pop();
+                    address.admin_area_2 = billingAddress.province.split('-').pop();
                 }
                 orderData.purchase_units[0].billing = {
                     address
